main.go

```diff
@@ -874,7 +874,7 @@ func configureSite(ctxt context.Context, ctx *cli.Context, siteNum int, siteStrs
 	defer globalConnStatsRWMutex.Unlock()
 	// reset connstats
 	globalConnStats = []*ConnStats{}
-	if len(endpoints) == 1 {
+	if len(endpoints) == 1 && ctx.GlobalBool("rr-dns-mode") {
```

Comment:
Maybe they are planning to expand to a cluster later?

Reply:
In that case it would be fine to always enable `rr-dns-mode` in a single cluster situation. I rather not have extra CLI options. Less is simpler and if it does the job, I rather not configure...